jax_unirep/featurize.py

```diff
@@ -39,7 +38,7 @@ def rep_same_lengths(
 
 
 def rep_arbitrary_lengths(
-    seqs: Iterable[str], params: Dict
+    seqs: Iterable[str], params: Dict, apply_fun, mlstm_size: str
```

Comment:
```suggestion
    seqs: Iterable[str], params: Dict, apply_fun: Callable, mlstm_size: int
```

`Callable` is importable from the `typing` module.

Reply:
good catch1